codejail_service/apps/api/v0/views.py

```diff
@@ -118,6 +124,12 @@ def code_exec(request):
     slug = params.get('slug')
     unsafely = params.get('unsafely')
 
+    # Help detect unusual traffic or filter out activity from certain sources
+    if limit_overrides_context:
+        set_custom_attribute('codejail.exec.limit_override', limit_overrides_context)
```

Comment:
Annotating would be a great way to explain what this is, or point to docs that explain.